sdk/eventhub/Azure.Messaging.EventHubs/src/EventHubsException.cs

```diff
@@ -201,7 +201,17 @@ public override string Message
         /// <returns>A <see cref="System.String" /> that represents this instance.</returns>
         ///
         public override string ToString() =>
-            $"{ typeof(EventHubsException).FullName }({ Reason }): { Message }{ Environment.NewLine }{ StackTrace }";
+            $"{ typeof(EventHubsException).FullName }({ Reason }): { Message }{ Environment.NewLine }{ StackTrace }{ FormatInnerException() }";
```

Comment:
I think [Exception.ToString](https://github.com/microsoft/referencesource/blob/master/mscorlib/system/exception.cs#L450) already does this. Why do we not just override Message?

Reply:
I don't recall why we didn't originally include the reason as a suffix added to the message.  In hindsight, that may have been a better idea.   At this point, I don't want to change the current format so I'm going to leave this as-is.